03-Raster-Calculations-In-R.Rmd

```diff
@@ -143,97 +172,78 @@ CHM_HARV <- DSM_HARV - DTM_HARV
 
 #plot the output CHM
 plot(CHM_HARV,
-     main="NEON Canopy Height Model - Subtracted\n Harvard Forest") 
+     main="NEON Canopy Height Model - Manual Subtract\n Harvard Forest") 
```

Comment:
Changed to `Canopy Height Model - Raster Math Subtract\n NEON Harvard Forest` since it is being compared to 
`Canopy Height Model - Overlay Subtract\n NEON Harvard Forest`

We can fix the NEON Harvard Forest once a figure title format is agreed on.